enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/IndexingTestSupport.scala

```diff
@@ -51,31 +54,50 @@ trait IndexingTestSupport extends ExecutionEngineFunSuite with CypherComparisonS
   }
 
   protected def assertSeekMatchFor(value: Value, nodes: Node*): Unit = {
-    val query = "CYPHER runtime=slotted MATCH (n:%s) WHERE n.%s = $%s RETURN n".format(LABEL, PROPERTY, PROPERTY)
+    val query = "MATCH (n:%s) WHERE n.%s = $%s RETURN n".format(LABEL, PROPERTY, PROPERTY)
     testIndexedRead(query, Map(PROPERTY -> value.asObject()), "NodeIndexSeek", nodes)
   }
 
   protected def assertScanMatch(nodes: Node*): Unit = {
-    val query = "CYPHER runtime=slotted MATCH (n:%s) WHERE EXISTS(n.%s) RETURN n".format(LABEL, PROPERTY)
+    val query = "MATCH (n:%s) WHERE EXISTS(n.%s) RETURN n".format(LABEL, PROPERTY)
     testIndexedRead(query, Map(), "NodeIndexScan", nodes)
   }
 
   protected def assertRangeScanFor(op1: String, bound1: Value, op2: String, bound2: Value, nodes: Node*): Unit = {
     val predicate1 = s"n.$PROPERTY $op1 $$param1"
     val predicate2 = s"n.$PROPERTY $op2 $$param2"
-    val query = s"CYPHER runtime=slotted MATCH (n:$LABEL) WHERE $predicate1 AND $predicate2 RETURN n"
+    val query = s"MATCH (n:$LABEL) WHERE $predicate1 AND $predicate2 RETURN n"
     testIndexedRead(query, Map("param1" -> bound1.asObject(), "param2" -> bound2.asObject()), "NodeIndexSeekByRange", nodes)
   }
 
   private def testIndexedRead(query: String, params: Map[String, AnyRef], wantedOperator: String, expected: Seq[Node]): Unit = {
-    val result = graph.execute(query, params)
+    if (cypherComparisonSupport) {
+      val result =
+        executeWith(
+          TestConfiguration(
+            Versions(Versions.V3_4, Versions.V3_3, Versions.Default),
+            Planners(Planners.Cost, Planners.Default),
+            Runtimes(Runtimes.Interpreted, Runtimes.Slotted, Runtimes.Default)
+          ),
+          query,
+          params = params,
+          planComparisonStrategy = ComparePlansWithAssertion(_ should useOperators(wantedOperator),
+                                                             expectPlansToFail = Configs.AllRulePlanners)
```

Comment:
` expectPlansToFail = Configs.AllRulePlanners` is not needed because you don't expect any RulePlanners to succeed so their plans will not be checked anyway